prime-router/src/main/kotlin/azure/DatabaseAccess.kt

```diff
@@ -85,7 +85,7 @@ typealias DataAccessTransaction = Configuration
  *
  * The companion object does the connection pooling and settings.
  */
-class DatabaseAccess(private val create: DSLContext) : Logging {
+class DatabaseAccess(val create: DSLContext) : Logging {
```

Comment:
I have a suspicion that running these queries in a transaction might be adding to the overhead as even the queries that return 100K+ results run at the worst in 10 seconds.  

Making this private meant that everything had to be done in a transaction or added a function to this class